python/ray/data/_internal/execution/interfaces.py

```diff
@@ -146,6 +147,14 @@ def input_dependencies(self) -> List["PhysicalOperator"]:
         ), "PhysicalOperator.__init__() was not called."
         return self._input_dependencies
 
+    def completed(self) -> bool:
```

Comment:
Since we have this interface now, can we update the docstring of `get_work_refs` and `has_next`? We were trying to explain this completion semantics there.

Reply:
Done.